### PR TITLE
CHK-440: Feat/pickup

### PR DESCRIPTION
**Pull request by jeffersontuc**
#### What problem is this solving?

The same as https://github.com/vtex-apps/checkout-graphql/pull/107

#### How should this be manually tested?

It should be tested alongside the PR in the above description.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by lucasecdb — DISMISSED** [review]

**Review by wdsrocha — DISMISSED** [review]

**Comment by jeffersontuc** [comment]
Rebased!

File: react/Mutations.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export default {
   estimateShipping,
   insertCoupon,
   selectDeliveryOption,
+  selectPickupOption,
```
Comment (lucasecdb) [inline react/Mutations.tsx]: we shouldn't update this file anymore

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: react/Mutations.tsx (hunk 1 above)
Comment (jeffersontuc) [reply react/Mutations.tsx]: oh, it's true!

Comment (jeffersontuc) [reply react/Mutations.tsx]: done.

**Review by lucasecdb — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]

**Pull request merged**